cockroachdb-in-comparison.md

```diff
@@ -2,3 +2,19 @@
 title: CockroachDB in Comparison
 toc: false
 ---
+
+| | CockroachDB | MySQL | PostgreSQL | Cassandra | HBase | MongoDB | Riak 
+--|--|--|--|--|--|--|--
+Automated Scaling | Yes | No | No | Yes | Yes  | Yes | Yes 
+Automated Failover & Repair | Yes | No | No | Yes | Yes | Yes | Yes 
```

Comment:
I'd just call this Automated Failover. The repair suggests that it is consistent.